dipy/segment/clustering.py

```diff
@@ -416,7 +417,7 @@ class QuickBundles(Clustering):
     metric : str or `Metric` object (optional)
         The distance metric to use when comparing two streamlines. By default,
         the Minimum average Direct-Flip (MDF) distance [Garyfallidis12]_ is
-        used and requires streamlines to have the same number of points.
+        used and streamlines are automatically resampled so they have 18 points.
```

Comment:
I would prefer this be a kwarg that users can easily change if they want to. Would that be hard to implement?